Add interactive capability for show command

icanhazstring opened the pull request:
This should resolve #233

Commit:
Add interactive capability for show.

Resolve #233

DanielBadura commented on src/Console/Command/ShowCommand.php:
```diff
@@ -40,16 +43,30 @@ public function __construct(Store $store, EventSerializer $serializer, Aggregate
-        $aggregate = InputHelper::string($input->getArgument('aggregate'));
-        $id = InputHelper::string($input->getArgument('id'));
+        $aggregate = InputHelper::nullableString($input->getArgument('aggregate'));
+        while ($aggregate === null) {
```
Isnt an `if` sufficient for both arguments?

icanhazstring replied:
If you hit just enter you will be asked again until you cancel the command, or enter something.

DanielBadura replied:
I'm not sure if this approach or a error message which hints the user that he needs to choose one option to proceed is better DX.

But lets keep it as is with the requestioning until a value is choosen, maybe with an additional text hinting the user that no input was given? This is up to you if you want to put more effort into it. 

A Testcase covering that the arguments are now optional would be great.

icanhazstring replied:
Good point, will add a hint and test 😉

Commit:
Add unit tests and remove while

Commit:
Use correct type after question

icanhazstring commented:
I removed the `while` statement as its not necessary. The question helper will do its part here.
Also added tests using `CommandTester`. If this is not wanted I can refactor it to be structured as the rest of the tests :)

Commit:
Use class-string for root registry

DanielBadura reviewed (approved):
This looks good, thank you @icanhazstring!
